site/content/entry/citra-progress-report-2019-q1.md

```diff
@@ -0,0 +1,269 @@
+`libZMQ` also proved to be very buggy and it seemed to be an unnecessarily big dependency for scripting.
+
+Unfortunately, [EverOddish](https://github.com/EverOddish) was very busy with other stuff and couldn't work on rewriting it.
+After waiting for a while, [wwylele](https://github.com/wwylele) took over and [reimplemented](https://github.com/citra-emu/citra/pull/4609) scripting over UDP using the [boost asio library](https://think-async.com/Asio/), thus entirely removing `libZMQ` dependency.
+
+### Status of Official Android App ([#4324](https://github.com/citra-emu/citra/pull/4324), [#4450](https://github.com/citra-emu/citra/pull/4450), [#4575](https://github.com/citra-emu/citra/pull/4575))
+
+When developer SachinVin unofficially ported Citra to Android, our own team members [BreadFish64](https://github.com/BreadFish64) and [jroweboy](https://github.com/jroweboy) were secretly working on an official port.
+Since the app was based on a few months old Citra code, it didn't contain many features, fixes, and performance improvements that were present in latest Citra code.
+This unoffical port, also heavily borrowed UI from Dolphin-emu's android app, which didn't fully meet the requirements of Citra.
+
+However, since then, [BreadFish64](https://github.com/BreadFish64) has been working along with [jroweboy](https://github.com/jroweboy) and [liushuyu](https://github.com/liushuyu), tirelessly to officially port Citra for Android.
+Although the app isn't ready for public usage, the work is progressing slowly but surely.
+If any of you out there have professional experience with Android app development, join us on our [discord](https://discordapp.com/invite/FAXfZV9) and let's collaborate.
+
+### [Flatpak Support](https://github.com/citra-emu/citra/pull/4383) by [bscubed](https://github.com/bscubed)
+
+As a result of collaboration between developers [Flame Sage](https://github.com/chris062689) and [bscubed](https://github.com/bscubed), we finally have [flatpak](https://flatpak.org/) support for both Citra nightly and Citra Canary builds.
+For the uninitiated, Flatpak is a system for building, distributing, and running sandboxed desktop applications on Linux.
+Flatpak support meant that Citra could now be installed and run on virtually any Linux distribution without running into OS specific issues.
+
+### Conclusion
+
+There's been [other exciting work from many different contributors](https://github.com/citra-emu/citra/pulls?page=1&q=is%3Apr+is%3Aclosed+merged%3A2018-09-14T00%3A00%3A00%2B00%3A00..2019-03-20T00%3A00%3A00%2B00%3A00+sort%3Aupdated-asc&utf8=%E2%9C%93), and we want to thank them all, but writing articles and blog posts is a very time consuming process, and we don't have anyone currently writing!
+If you would like to contribute to the project by joining the writing staff, reach out on the forums or on [Discord](https://discordapp.com/invite/FAXfZV9) and show us what you can do.
```

Comment:
I would say `Discord` should link to `https://citra-emu.org/discord` instead of a direct invite.